CHANGES

```diff
@@ -36,6 +36,7 @@ Version 8.9
 - Added weekly report emails behind feature flag.
 - Rebrand Rules as Alerts / Alert Rules.
 - Add frequency to Alerts.
+- Send email verification to specific emails
```

Comment:
This should be expanded a bit more. It's not very clear what this means as an end user reading this. tbh the git commit message is more descriptive than this. :)